apps/extension/src/ui/domains/Staking/useStakingBanner.tsx

```diff
@@ -75,53 +77,41 @@ const nomPoolEligibleAddressBalancesState = selector({
   },
 })
 
-const NOM_POOL_STAKED_CACHE: Record<string, Promise<ResponseNomPoolStake>> = {}
-
-const getNomPoolStakedBalance = ({
-  chainId,
-  addresses,
-}: {
-  chainId: ChainId
-  addresses: Address[]
-}): Promise<ResponseNomPoolStake> => {
-  const cacheKey = `${chainId}::${addresses.join("-")}`
-
-  if (!NOM_POOL_STAKED_CACHE[cacheKey]) {
-    NOM_POOL_STAKED_CACHE[cacheKey] = api.getNomPoolStakedBalance({
-      chainId,
-      addresses,
-    })
-  }
-
-  return NOM_POOL_STAKED_CACHE[cacheKey]
-}
-
 const nomPoolStakedBalancesState = selector({
   key: "nomPoolStakedBalancesState",
   get: async ({ get }) => {
     const nomPoolEligibleAddressBalances = get(nomPoolEligibleAddressBalancesState)
     const eligibleChains = Object.keys(nomPoolEligibleAddressBalances)
-
-    const result: Record<ChainId, ResponseNomPoolStake> = {}
+    type AddressNomPoolStake = Record<string, null | bigint>
+    const result: Record<ChainId, AddressNomPoolStake> = {}
 
     if (eligibleChains.length === 0) return {}
+    const balances = get(ownedBalancesState)
 
-    const nomPoolPromises = NOM_POOL_SUPPORTED_CHAINS.map(async (chainId) => {
+    NOM_POOL_SUPPORTED_CHAINS.map((chainId) => {
       const eligibleAddressesBalances = nomPoolEligibleAddressBalances[chainId]
       const addresses = eligibleAddressesBalances ? Object.keys(eligibleAddressesBalances) : []
       if (addresses.length == 0) return
 
-      try {
-        result[chainId] = await getNomPoolStakedBalance({
-          chainId,
-          addresses,
-        })
-      } catch (err) {
-        Sentry.captureException(err, { tags: { chainId } })
-      }
-    })
+      result[chainId] = balances
```

Comment:
@alecdwm lmk if you think this is a good way to determine the presence of a nom pool staking balance